tests/spanner_dbapi/test_connect.py

```diff
@@ -0,0 +1,115 @@
+# Copyright 2020 Google LLC
+#
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file or at
+# https://developers.google.com/open-source/licenses/bsd
+
+"""connect() module function unit tests."""
+
+import unittest
+from unittest import mock
+
+
+def _make_credentials():
+    import google.auth.credentials
+
+    class _CredentialsWithScopes(
+        google.auth.credentials.Credentials, google.auth.credentials.Scoped
+    ):
+        pass
+
+    return mock.Mock(spec=_CredentialsWithScopes)
+
+
+class Testconnect(unittest.TestCase):
+    def _callFUT(self, *args, **kw):
+        from spanner_dbapi import connect
```

Comment:
Why do you need to bury the import here instead of just doing `import spanner_dbapi` above?

Reply:
I don't like it myself, and when I first saw it, it made me confused. But that is what they do in Google Cloud ([example](https://github.com/googleapis/python-spanner/blob/45a15382bc1e62dedc944f6484c15ba929338670/tests/unit/test_database.py#L208-L211)). I think it's a way to avoid using instances patched with mocks by another tests. If it's okay to use a single import at the top of the file, I'm definitely for it.

Reply:
That's interesting, I see the python-spanner tests bury a lot of imports, including things (like the proto in your example) that you wouldn't really expect others to mock or to have import-time effects.

Maybe there's a good reason to do this, but the test runs fine without it. I think in general it's better to have code that's easy to understand, and if we find out that this doesn't work we can copy the pattern from python-spanner.

FWIW `_callFUT` goes all the way back to the first public release of python-spanner (https://github.com/googleapis/python-spanner/commit/fb9733b), I don't see any justification for using it in PRs or git history.

Reply:
SG, fixed all of it